corehq/apps/case_importer/templates/case_importer/partials/ko_import_history.html

```diff
@@ -82,6 +88,18 @@
             <!--/ko-->
             <!--/ko-->
         </td>
+        {% if request|toggle_enabled:"SUPPORT" %}
+            <td>
+                <a data-bind="attr: {href: formIdsUrl()}" target="_blank">
```

Comment:
I don't think the `target="_blank"` will do anything since this is a file download.

Reply:
It's actually not a file download despite what the URL makes you think (`Content-Disposition` isn't set on those links. The downloadUrl also sends you to an interstitial page which then sends you to the download, but I didn't fix that since I felt it was out of scope).

Reply:
Ah, weird - yeah, I saw `content_type='text/plain'` in the view and thought it was a download.